### PR TITLE
Feat/568 review page tests

### PR DESCRIPTION
**Pull request by mlauren**
## Description
Adds test suite for review page functionality. Adds additional tweak to individual to register field change when page is indexed in state.

## Original issue(s)

department-of-veterans-affairs/va-forms-system-core#568 

## Testing done
✅ except for checkbox group validation that should be handled for #238

## Screenshots
Screen Shot 2022-08-22 at 3 42 16 PM

## Definition of done

- [ ] Package.json version has been updated to match Github release tag
- [x] Documentation has been updated, if applicable
- [x] A link to the original github issue has been provided
- [x] No sensitive information (i.e. PII/credentials/internal URLs etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by bjmccotter7192 — COMMENTED** [review]
Just a few questions but otherwise nice work on those tests!

File: examples/form-styles.css [hunk 1]
```diff
@@ -7,20 +7,20 @@ form [type=submit] {
 }
 
 /* Review Page */
-.review-page--page-heading--text {
+.review-page__page-heading--text {
```
[the remark this answers is not in the document]
Comment (mlauren) [reply examples/form-styles.css:10]: I'm just going to check out these changes so they can be addressed in another ticket. #561

File: src/form-builder/CheckboxField.tsx [hunk 2]
```diff
@@ -24,6 +24,7 @@ import { gatherFieldData, PageContext } from '../form-data';
  */
 const CheckboxField = (props: CheckboxProps): JSX.Element => {
   const { listOfPages, setListOfPages, currentPath } = useContext(PageContext);
+  const currentPage = listOfPages.find((page) => page.path === currentPath);
```
[the remark this answers is not in the document]
Comment (mlauren) [reply src/form-builder/CheckboxField.tsx:27]: yes

File: test/form-builder/CheckboxFieldGroup.test.tsx [hunk 3]
```diff
@@ -59,7 +59,7 @@ describe('Form Builder - CheckboxFieldGroup', () => {
     expect(firstCheckbox?.label).toEqual('Eggs');
   });
 
-  test('renders the default "required" validation error message', async() => {
+  test.skip('renders the default "required" validation error message', async() => {
```
[the remark this answers is not in the document]
Comment (mlauren) [reply test/form-builder/CheckboxFieldGroup.test.tsx:62]: yes, yesterday it was not merged

**Pull request merged**